.editorconfig

```diff
@@ -8,3 +8,8 @@ charset = utf-8
 insert_final_newline = true
 indent_style = space
 indent_size = 4
+max_line_length = 100
```

Comment:
I think we still aim at 80 lines, with hard limit at 100. Should probably be set to 80 here,l